plugins/CoreHome/javascripts/popover.js

```diff
@@ -142,7 +142,17 @@ var Piwik_Popover = (function () {
 
         /** Set the title of the popover */
         setTitle: function (titleHtml) {
+            var titleText = $('<div>' + titleHtml + '</div>').text();
+            if (titleText.length > 60) {
+                titleHtml = $('<span>').attr('class', 'tooltip').attr('title', titleText).html(titleHtml);
```

Comment:
> Just tried it using `titleHtml='"%<b>stt&7ä'` and it converts it to `"%stt&7ä` so some information gets lost

I assumed (due to the name of the variable) that it is possible to set html in titles. If that isn't or shouldn't be possible, we could simply use the content given to the method instead of filtering the html.